MAUI/Popup/popup-size.md

```diff
@@ -0,0 +1,333 @@
+            listView.WidthRequest = 350;
+            listView.ScrollBarVisibility = ScrollBarVisibility.Never;
+            listView.ItemTemplate = new DataTemplate(() =>
+            {
+                ViewCell viewCell = new ViewCell();
+                var grid = new Grid() { RowSpacing = 1 };
+                grid.ColumnDefinitions.Add(new ColumnDefinition() { Width = 60 });
+                grid.ColumnDefinitions.Add(new ColumnDefinition() { Width = 200 });
+                grid.ColumnDefinitions.Add(new ColumnDefinition() { Width = 20 });
+                var contactImage = new Image()
+                {
+                    VerticalOptions = LayoutOptions.Center,
+                    HorizontalOptions = LayoutOptions.End,
+                    HeightRequest = 40
+                };
+                contactImage.SetBinding(Image.SourceProperty, new Binding("ContactImage"));
+                var contactName = new Label()
+                {
+                    VerticalTextAlignment = TextAlignment.Center,
+                    HorizontalTextAlignment = TextAlignment.Center,
+                    LineBreakMode = LineBreakMode.NoWrap,
+                    FontSize = 16,
+                };
+                contactName.SetBinding(Label.TextProperty, new Binding("ContactName"));
+                var contactType = new Image()
+                {
+                    VerticalOptions = LayoutOptions.Center,
+                    HorizontalOptions = LayoutOptions.End,
+                    HeightRequest = 15,
+                };
+                contactType.SetBinding(Image.SourceProperty, new Binding("ContactType"));
+                grid.Children.Add(contactImage);
+                Grid.SetColumn(grid, 0);
+                grid.Children.Add(contactName);
+                Grid.SetColumn(grid, 1);
+                grid.Children.Add(contactType);
+                Grid.SetColumn(grid, 2);
+                viewCell.View = grid;
+                return viewCell;
+            });
+            viewModel = new ContactsViewModel();
+            listView.ItemsSource = viewModel.contactsinfo;
+            popup = new SfPopup();
+            popup.HeaderTitle = "ListView";
+            popup.HeightRequest = 320;
+            popup.ContentTemplate = new DataTemplate(() =>
+            {
+                return listView;
+            });
+            StackLayout stackLayout = new StackLayout();
+            Button isOpenButton = new Button();
+            isOpenButton.Clicked += heightWidth_Clicked;
+            isOpenButton.Text = "Click to show popup";
+            stackLayout.Children.Add(isOpenButton);
+            this.Content = popup;
+        }
+
+        private void heightWidth_Clicked(object sender, EventArgs e)
+        {
+			popup.Show();
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Popup with HeightRequest and WidthRequest](Images/popup-size//maui-popup-with-heightrequest-and-widthrequest.png)

```

Comment:
.NET MAUI Popup

change as .NET MAUI Popup wherever used Popup (in all image alt text)